src/components/InputBlock/InputBlock.jsx

```diff
@@ -1,82 +0,0 @@
-/* eslint-disable react/destructuring-assignment */
-/* eslint-disable react/forbid-prop-types */
-import React from 'react';
-import PropTypes from 'prop-types';
-// import Input from '@material-ui/core/Input';
-import OutlinedInput from '@material-ui/core/OutlinedInput';
-import TextField from '@material-ui/core/TextField';
-// import InputLabel from '@material-ui/core/InputLabel';
-import InputAdornment from '@material-ui/core/InputAdornment';
-import { Typography } from '@material-ui/core';
-
-class InputBlock extends React.Component {
-  constructor(props) {
-    super(props);
-    this.caption = props.caption;
```

Comment:
I do not see any reason to copy props to class properties.
And also all required for component data should be kept in component `state`.